src/Features/Core/Portable/CodeRefactorings/AbstractRefactoringHelpersService.cs

```diff
@@ -410,6 +413,18 @@ protected virtual IEnumerable<SyntaxNode> ExtractNodesSimple(SyntaxNode? node, I
             {
                 yield return node.Parent;
             }
+
+            // If there is only one variable declared in the field, then consider the only field is selected
+            // e.g.
+            // class Bar
+            // {
+            //     pub$$lic int i;
+            // }
+            // Treat as 'i' is selected
+            if (TryGetVariableDeclaratorInSingleFieldDeclaration(node, out var singleVariable))
+            {
+                yield return singleVariable;
```

Comment:
interesting.  we might want to doc that the node returned may not even intersect the position with it's fullspan.

Reply:
As Andrew points above, the initlizer is also needed to be considered. So I extract the method to a separate one. No need to update the comments here